LipematWordpress/Sniffs/Sniff.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * WP phpcs Coding Standard.
+ *
+ * @package lipemat/wp-phpcs
+ * @link https://github.com/lipemat/wp-phpcs
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace LipematWordpress\Sniffs;
```

Comment:
While the library is `wp-phpcs` these custom sniffs are technically not WordPress specific and could be used on any JS/React project. The `Lipe/Sniffs` namespace follows the pattern of the other "lipemat" libraries.
Also, the directory should be named 'src' not "LipematWordPress".

Reply:
I realized that phpcs is using the folder name where the `ruleset.xml` is located, for example, if we include the custom sniff with the directory name `src` and run `phpcs -i` to see the list of standard sniffers, you are going to get this:

```
The installed coding standards are MySource, PEAR, PSR1, PSR12, PSR2, Squiz, Zend, WordPress-VIP-Go, WordPressVIPMi
nimum, PHPCompatibility, PHPCompatibilityParagonieRandomCompat, PHPCompatibilityParagonieSodiumCompat, PHPCompatibi
lityWP, VariableAnalysis, WordPress, WordPress-Core, WordPress-Docs, WordPress-Extra and src
```

As you can see in the list, the last name in the list is our standard sniff name `src`, which can be a confusing name, I think we can use the directory `src/Lipe/Sniffs`, that way the standard name that going to be used will be `Lipe`.

Let me know what do you think.

Reply:
👍